src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx

```diff
@@ -1,30 +1,40 @@
 import styled from "@emotion/styled";
 import tw from "twin.macro";
-import { mixin } from "../../../styles";
 
 export const ViewerContainer = styled.div`
   ${tw`text-center p-6 flex flex-col justify-center rounded-xl min-h-400 lg:min-h-600`}
+  border-width: 2px;
+  border-style: dashed;
 
   &.default {
-    ${mixin.dropzoneStatus({ bg: "#f3f8fc", borderColor: "#0099cc" })}
+    background-color: #f3f8fc;
+    box-shadow: 0 0 0 8px #f3f8fc;
+    border-color: #0099cc;
   }
 
   &.accept {
-    ${mixin.dropzoneStatus({ bg: "#f5fbf7", borderColor: "#00c04a" })}
+    background-color: #f5fbf7;
+    box-shadow: 0 0 0 8px #f5fbf7;
+    border-color: #00c04a;
   }
 
   &.warning {
-    ${mixin.dropzoneStatus({ bg: "#ffe600", borderColor: "#ffbf00" })}
+    background-color: #ffe600;
+    box-shadow: 0 0 0 8px #ffe600;
+    border-color: #ffbf00;
 
     .unverified-btn {
       ${tw`transition-colors duration-200 ease-out border border-solid border-yellow-600 text-white bg-yellow-600 py-2 px-6 rounded font-medium text-center align-middle min-w-135 cursor-pointer hover:border-yellow hover:bg-yellow hover:shadow-md hover:no-underline`}
     }
   }
 
   &.invalid {
+    background-color: #f7d7d7;
+    box-shadow: 0 0 0 8px #f7d7d7;
+    border-color: #e46767;
+
     ${tw`text-red-400`}
```

Comment:
all these "background-color: #...." is the color defined in our tailwind config? if so we can use the tailwind class like "text-red-400".

Reply:
updated, only box-shadow abit harder to share variables, defining them in the tw config file for now, not sure if good solution not 😬